src/main/java/io/getunleash/metric/JavaHttpMetricsSender.java

```diff
@@ -25,7 +23,7 @@ public class UnleashMetricsSender {
     private final URL clientRegistrationURL;
     private final URL clientMetricsURL;
 
-    public UnleashMetricsSender(UnleashConfig unleashConfig) {
+    public JavaHttpMetricsSender(UnleashConfig unleashConfig) {
```

Comment:
Very optional nit: Happy with the name here? The Java feels a bit redundant in a Java program. DefaultMetricsSender maybe?

Reply:
Well, it is using java.net.http :P But agreed that it feels redundant. Default is fair. I'll do the rename